.github/workflows/test.yml

```diff
@@ -71,10 +70,20 @@ jobs:
         # LD_LIBRARY_PATH doesn't have /usr/local/lib on it by default
         # like you might expect because we're running as root, so we
         # just append it (it's so we can find python itself.)
-      - name: Test
+
+      - name: Configure environment
+        run: |
+          echo "PXR_PLUGINPATH_NAME=$GITHUB_WORKSPACE/build/dist/resources/plugInfo.json" >> "$GITHUB_ENV"
```

Comment:
Unsure if this is a problem that will show up, but the USD in Katana, this will be called `FNPXR_PLUGINPATH_NAME`.

Reply:
Ah yeah I remember from prototyping this with Katana. There are other changes required to make it work with Katana. Though not too many. I.e. I needed to set `KATANA_API_LOCATION`, add the location of Katana's `Support.cmake` to `CMAKE_MODULE_PATH`,  then modify our `CMakeLists.txt` 

```
if (KATANA_API_LOCATION)
    set(USE_KATANA_THIRDPARTY_LIBS ON)
    include(Support)
else()
    find_package(pxr REQUIRED)
endif ()
```

Reply:
Ok cool, you're aware of it. Just wanted to make sure it was on your internal radar.